pkg/storage/stores/shipper/bloomshipper/client.go

```diff
@@ -29,6 +28,14 @@ const (
 	fileNamePartDelimiter = "-"
 )
 
+type BoundsCheck uint8
```

Comment:
I'd love to keep this in the `v1` lib so I can use it there as well (can't import this package there)